dart/lib/src/sentry_client.dart

```diff
@@ -181,7 +181,7 @@ class SentryClient {
 
     for (final processor in eventProcessors) {
       try {
-        event = await processor(event, hint);
+        event = await processor(event.copyWith(), hint);
```

Comment:
do we need this if all the immutability is fixed?

Reply:
yes, because, as the event members are public, it's the only way to guarantee that nothing is immutable in the event passed to the processor

Reply:
`copyWith` this should be called to `beforeSend` too then?

Reply:
and every other callback that could be set by the client

Reply:
or in the first method of the proessing pipeline, right after captureEvent maybe?

Reply:
✅ I finally clone the event in the beginning of the `client.captureEvent`